lib/fetchResource.js

```diff
@@ -13,7 +13,7 @@ const upgradeToHttps = require('./upgradeToHttps');
 /**
  * Download the resource
  *
- * @param {string} resourceUrl
+ * @param {string} resourceUrl The URL to fetch
```

Comment:
Looks good, but note how other JSDoc comments seperate the variable name and the description with a dash (`-`)